spring-cloud-dataflow-rest-client/src/test/java/org/springframework/cloud/dataflow/rest/client/DataflowTemplateTests.java

```diff
@@ -86,6 +86,25 @@ public void testDataFlowTemplateContructorWithNonExistingUri() throws URISyntaxE
 		new DataFlowTemplate(new URI("http://doesnotexist:1234"));
 	}
 
+	@Test
+	public void testThatObjectMapperGetsPrepared() {
+		final ObjectMapper objectMapper = new ObjectMapper();
+		DataFlowTemplate.prepareObjectMapper(objectMapper);
+		assertCorrectMixins(objectMapper);
+	}
+
+	@Test
+	public void testPrepareObjectMapperWithNullObjectMapper() {
+		try {
+			DataFlowTemplate.prepareObjectMapper(null);
+		}
+		catch (IllegalArgumentException e) {
+			assertEquals("The objectMapper must not be null.", e.getMessage());
+			return;
+		}
+		fail("Expected an IllegalArgumentException to be thrown.");
```

Comment:
We can move the fail inside the try block.  To be consistent with the other changes.